MainUI.jsx

```diff
@@ -60,6 +87,10 @@ class MainUI extends React.Component {
     this.props.deckVisible.mutate(mutation);
   }
 
+  handleBaseStatsToggle = () => {
+    this.setState({ showBaseStats: !this.state.showBaseStats });
```

Comment:
I would handle this via props like deckVisibility above (if it's not clear from the code, mutation is a mapping from current state to new state).  This will make it get synchronized through React.   The main change is to add it to `propTypes` above and modify `withStorage` below to define deserialization/default values.

Reply:
sure will do.

I had thought about making it a prop, but I was concerned that this was the type of state that we didn't actually want to serialize.  My worry is that a user would turn on the stats locally, and that suddenly all users  (via the magic of firebase) would have this choice about how a specific user wanted their UI to look reflected onto their layout.  I'm not 100% what is up with the property storage, however, so I might be misreading it.

Reply:
Yeah, the property storage stuff is setup to share across all users using the same game UUID, so all windows viewing the same game would see the stats toggle if we wire it through storage.  I could go either way on it being persisted, but right now I believe we persist everything except the menu visibility to all instances.